src/pages/home/owners/OwnersTable.tsx

```diff
@@ -125,7 +123,7 @@ export default function OwnersTable({
               index
             ) => (
               <TableRow key={ id || index }>
-                <TableCell>
+                <TableCell sx={ { display: { xs: "none", sm: "table-cell" } } }>
```

Comment:
Realized that the name is not always going to be available. The song count and email will be though so I changed the table to display those 2 on smaller screens